README.md

```diff
@@ -5,25 +5,29 @@
 
 ## TimescaleDB
 
-TimescaleDB is an open-source database designed to make SQL scalable for
-time-series data. It is engineered up from PostgreSQL, providing automatic
-partitioning across time and space (partitioning key), as well as full
-SQL support.
-
-TimescaleDB is packaged as a PostgreSQL extension.  All code is licensed
-under the Apache-2 open-source license, with the exception of source code
-under the `tsl` subdirectory, which is licensed under
-the [Timescale License (TSL)](https://github.com/timescale/timescaledb/blob/master/tsl/LICENSE-TIMESCALE).
-For clarity, all code files reference licensing in their header.  [Contributors welcome.](https://github.com/timescale/timescaledb/blob/master/CONTRIBUTING.md)
+TimescaleDB is an open-source database designed to make SQL scalable for time-series data. 
+It is engineered up from PostgreSQL and packaged as a PostgreSQL extension, providing automatic 
+partitioning across time and space (partitioning key), as well as full SQL support.
+
+[Timescale Cloud](https://tsdb.co/GitHubTimescaleCloud) is our fully managed, hosted version of 
+TimescaleDB, available in the cloud of your choice (pay-as-you-go, $300 in credits to start).
+To determine which option is best for you, see [Timescale Products](https://tsdb.co/GitHubTimescaleProducts) 
+for more information about our Apache-2 version, TimescaleDB Community (self-hosted) and Timescale 
+Cloud (hosted), including: feature comparisons, FAQ, documentation, and support.
+
+Below is an introduction to TimescaleDB. For more information, please check out 
+these other resources:
+- [Developer Documentation](https://tsdb.co/GitHubTimescaleDocs)
+- [Slack Channel](https://slack-login.timescale.com)
+- [Timescale Release Notes & Future Plans](https://tsdb.co/GitHubTimescaleReleaseNotes)
 
+For reference and clarity, all code files in this repository reference licensing in their 
+header (either Apache-2-open-source license or [Timescale License (TSL)](https://github.com/timescale/timescaledb/blob/master/tsl/LICENSE-TIMESCALE)). 
```

Comment:
Sorry to be picky, but I think the official name is "Apache License, Version 2.0" and I know that it can be quite a sensitive issue on how you use names.

Reply:
Not picky at all - we should be accurate (I just used what we'd listed previously), but please adjust as appropriate!